DXRCore/DeusEx/Classes/DXRActorsBase.uc

```diff
@@ -1484,6 +1484,21 @@ function DebugMarkKeyPosition(vector pos, coerce string id)
     DebugMarkKeyActor(a, id);
 }
 
+static function bool IsNonlethal(coerce string damageType)
+{
+    return (damageType == "Stunned") || (damageType == "KnockedOut") || (damageType == "Poison") || (damageType == "PoisonEffect");
+}
+
+// returns true if the damage type can knock the pawn out
+static function bool CanKnockUnconscious(ScriptedPawn sp, coerce string damageType)
+{
+    if (#defined(injections)) {
+        return sp.mass > 5.0 && IsNonlethal(damageType) && (Animal(sp) != None || IsHuman(sp.class));
```

Comment:
I think this could be simplified a little bit
```suggestion
        return sp.mass > 5.0 && IsNonlethal(damageType) && Robot(sp) == None;
```

Reply:
Yes, but that assumes this will never be called on a pawn that isn't a Robot or Animal or Human. Probably true, but this is safer.